content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -0,0 +1,413 @@
+benefit for users sending bulk telemetry data would outweigh the cost of
+additional compute resources taken by the encoding. We are especially interested
+in this metric on the exporter side of the bridge, because it impacts the total
+cost of observability for users sending telemetry to a service provider. Here,
+we compare the cost of sending standard OTLP data with the cost of sending
+OTel-Arrow data.
+
+While the OTel-Arrow exporter uses somewhat more memory than its counterpart
+OTLP exporter, the difference is not a substantial driver of cost because other
+factors, including garbage collection, make the two exporters require similar
+amounts of memory and because memory costs 10% of CPU cost for the machine size
+tested.
+
+We are interested in comparing the cost and number of vCPU/hours spent in the
+two exporter configurations as well as the number of GiB/hour of telemetry sent.
+It is common in cloud computing environments that the price of one vCPU/hour is
+less than the cost of one GiB of egress to the public internet. We estimate that
+typically, one vCPU/hour costs typically one quarter to one half the cost of one
+GiB of egress.
+
+According to this logic, the OTel-Arrow bridge makes economic sense as we
+compare the difference in vCPU/hour with the difference in GiB exported. For
+example, in one trial the OTel-Arrow exporter used an average 77.0 vCPU and
+exported at a rate of 107 GiB/hour. In a similar trial with a similar quantity
+of aggregate data, we observe the OTLP exporter used 53.7 vCPU and exported 143
+GiB/hour. In most cloud compute agreements, the cost of 23 vCPU/hour is
+substantially less than the cost of 36 GiB/hour, so adopting an OpenTelemetry
+with Apache Arrow bridge will lower the total cost of telemetry transport.
+
+## Summary
+
+We reported on the performance of a number of OTel-Arrow exporter
+configurations, in each case offering substantial compression benefits compared
+with the OpenTelemetry OTLP encoding and without sacrificing reliability or
+scalability. Using ServiceNow Cloud Observability’s internal traces and logs
+data, OTel-Arrow achieves compression reduction of 15 to 30 times smaller than
+uncompressed data, generally offering 30% to 50% less network bandwidth than the
+OTLP protocol.
+
+This is just a beginning for the OpenTelemetry Protocol with Apache Arrow
+project. We expect to extend the benefits of Apache Arrow record batch
+processing further into the OpenTelemetry Collector pipeline. We look forward to
+an end-to-end OpenTelemetry Protocol with Apache Arrow pipeline with
+substantially lower overall costs, where OpenTelemetry SDKs can produce this
+format directly and telemetry processors can be written based on Apache Arrow
+libraries.
+
+More than that, we have shown that an OTel-Arrow bridge will lower overall
+telemetry costs today. The OpenTelemetry Protocol with Apache Arrow is ready for
+production use.
```

Comment:
Do you also want to invite people to contribute to the project? Or do you want them to try it out themselves and report about their own experience? Any "Call To Action"?

Reply:
I've added to the final paragraph:

```
We invite users and telemetry
systems to adopt OpenTelemetry Protocol with Apache Arrow components
for lower-cost telemetry transport.  If you find this interesting and
want to see more integration between OpenTelemetry and Apache Arrow,
please join our community or post an issue in our [project
repository](https://github.com/open-telemetry/otel-arrow).```

@lquerel and @svrnm let me know how this sounds. Thanks.